src/components/views/auth/CountryDropdown.tsx

```diff
@@ -41,15 +39,26 @@ function countryMatchesSearchQuery(query, country) {
     return false;
 }
 
+interface IProps {
+    value?: string;
+    onOptionChange: (country: PhoneNumberCountryDefinition) => void;
+    isSmall: boolean;
```

Comment:
```suggestion
    isSmall: boolean; // if isSmall, show +44 in the selected value
```

Would be good to keep the comment that was here, given that `isSmall` isn't nearly descriptive enough